components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/resources/publisher-api.yaml

```diff
@@ -6120,6 +6122,7 @@ paths:
       security:
         - OAuth2Security:
             - apim:api_publish
+            - apim:api_import_export
```

Comment:
Since this is for API Products, the scope should be **apim:api_product_import_export**